harness/harness-core-ui
# feat: [CDS-77809]: added split pane view in console logs for better error/suggestions readability

### Summary

- Added split pane in console logs for better readability of error and suggestions
- base height of the pane remains as it is `132px` and can be extended till `300px`

Manually Tested Scenarios :-  

- Presence of split pane in console view with large error message and suggestions
- No split pane should be present in the step-details view
- No split pane should be present in case of no error is present

#### Screenshots


https://github.com/harness/harness-core-ui/assets/106532291/7951b412-4acd-4580-9730-7c6fe49ce753

Coverage :-    

![image](https://github.com/harness/harness-core-ui/assets/106532291/e47d5b29-8997-466f-b749-a783b2add8fa)




#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)
